db/migrate/20220907145552_clean_trends_and_foreman_docker_data.rb

```diff
@@ -0,0 +1,14 @@
+require_dependency File.expand_path('cleanup_helper', './lib/tasks/')
+
+include CleanupHelper
```

Comment:
I don't like including a module. Feels like it ends up in the global namespace. Can't it be a class with only static methods?